cmd/justification/main.go

```diff
@@ -63,13 +64,32 @@ func realMain(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to setup kms client: %w", err)
 	}
+	defer func(kmsClient *kms.KeyManagementClient) {
```

Comment:
Go has an [io.Closer](https://pkg.go.dev/io#Closer) interface, so it might be cleaner to do something like:

```go
func gracefulClose(logger *log.Logger, c io.Closer) {
  if err := c.Close(); err != nil {
    logger.Errorf("failed to close: %s", err)
  }
}
```

and then just `defer` in a single line:

```go
defer gracefulClose(logger, kmsClient)
defer gracefulClose(logger, firestoreClient)
```

Reply:
done